### PR TITLE
feat:Added text,icon with badge, avatar with text

### PR DESCRIPTION
**Pull request by AnuragSahu11**
Capture
Capture2

*Added text with badge and icon with badge
*Added avatar with text
*made some refactoring to css variables

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -10,9 +10,9 @@
   font-size: 4.4px;
 }
 .avatar-xs img {
-  width: 30px;
-  height: 30px;
-  border-radius: 50%;
+  width: var(--l);
```
Comment (junaidshaikh-js) [inline components/avatar.css:13]: variable names should be something meaningful

File: components/avatar.css [hunk 2]
```diff
@@ -83,5 +85,53 @@
   height: 2em;
   width: 2em;
   background-color: var(--primary);
-  border-radius: 50%;
+  border-radius: var(--border-radius-circle);
+}
+
+/* avatar with text */
+
+.avatar-with-text {
```
Comment (junaidshaikh-js) [inline components/avatar.css:93]: we can shorten classname here `avatar-txt` 

no need to add `with`

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

**Review by GuptaEesh — COMMENTED** [review]
Looks good to me except that you could have used rems for values>10px.

**Pull request merged**